packages/drivers/odsp-urlResolver/api-report/odsp-urlresolver.api.md

```diff
@@ -4,10 +4,19 @@
 
 ```ts
 
+import { IContainerPackageInfo } from '@fluidframework/driver-definitions';
 import { IRequest } from '@fluidframework/core-interfaces';
 import { IResolvedUrl } from '@fluidframework/driver-definitions';
 import { IUrlResolver } from '@fluidframework/driver-definitions';
 
+// @internal (undocumented)
+export class FluidAppOdspUrlResolver implements IUrlResolver {
```

Comment:
@jatgarg Can you recommend documentation for this? Seems like a good opportunity to fill in some docs while we're moving this.

Reply:
You can put something like, 
This resolver is used to help resolve urls which belongs to onenote, office.com etc.

Reply:
Thanks!

Reply:
Updating accordingly

Reply:
I updated the documentation at ts file and rebuild it. The api report's `(undocumented)` mark is gone but I don't see the documentation here. Is that expected?

Reply:
Seems like expected. Resolving this one...